ci.env

```diff
@@ -0,0 +1,317 @@
+# You can leave this on "local". If you change it to production most console commands will ask for extra confirmation.
```

Comment:
A better option instead of this would be to just update the `.env.example` and assume it contains the CI variables. Having separate files obviously means both files need to be maintained.

Reply:
Yes, but the example file is for users. It works when people download the files without changing anything and run the Docker commands.